README.md

```diff
@@ -18,8 +18,8 @@ repositories {
 3. Add the dependency under ```dependencies```
 ```
 implementation 'com.rudderstack.android.sdk:core:1+'
```

Comment:
Can we update the version of the Rudder Android SDK from the prefix style to Maven-style version range ?

You can refer [here](https://docs.gradle.org/current/userguide/single_versions.html)

Reply:
As publishing to Maven Central prefers Maven-style version range over prefix style version range